src/pages/Login/Login.tsx

```diff
@@ -16,15 +16,19 @@ import InputAdornment from "@mui/material/InputAdornment";
 import { Visibility } from "@mui/icons-material";
 import { VisibilityOff } from "@mui/icons-material";
 import IconButton from "@mui/material/IconButton";
+import { NULL } from "sass";
 const Login = () => {
-  const { setCurrentUser } = useContext(AuthContext);
+  const { currentUser,setCurrentUser } = useContext(AuthContext);
   const navigate = useNavigate();
 
   const [email, setEmail] = useState<string>("");
   const [password, setPassword] = useState<string>("");
   const [loading, setLoading] = useState<boolean>(false);
   const [error, setError] = useState<string>("");
  const [showPassword,setShowPassword]=useState<boolean>(false);
+  if(currentUser!=null){
```

Comment:
Put this inside a `useEffect` and include `currentUser` in the dependency array.

Reply:
done